cloudigrade/account/management/commands/add_account.py

```diff
@@ -29,9 +29,13 @@ def handle(self, *args, **options):
         arn = options['arn']
         account_id = aws.extract_account_id_from_arn(arn)
 
-        # TODO AWS account permissions verification goes here.
-        found_instances = aws.get_running_instances(arn)
-        Account(account_arn=options['arn'], account_id=account_id).save()
-
-        self.stdout.write(self.style.SUCCESS(_('ARN Info Stored')))
-        self.stdout.write(_(f'Instances found include: {found_instances}'))
+        is_verified = aws.verify_account_access(arn)
+        if is_verified:
```

Comment:
why not just `if aws.verify_account_access(arn)`, it doesn't look like we use `is_verified` anywhere else.